src/Dawid/Fsd.Dawid.CS/Fsd.Dawid.Cs.Ex1/Fsd.Dawid.Cs.Ex1/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Dawid.Cs.Ex1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Excercise 1 - Introduction");
+            Console.WriteLine("---------------------------");
+            Console.WriteLine("Today is " + DateTime.Now.ToString());
+            Console.WriteLine();
+            Console.WriteLine(args.Length == 0 ? "You didn't specify arguments :(" : "Console arguments:");
+            Console.WriteLine();
+            for (int i=0; i< args.Length; i++)
```

Comment:
dobrze zrobić pustą linię przed innym blokiem kodu, tzn Console.WriteLine sie kończy, zaczyna się for -> tu powinna być przerwa